css-borders-4/Overview.bs

```diff
@@ -617,16 +617,26 @@ Initial: none
 Applies to: all elements
 Inherited: no
 Percentages: N/A
-Computed value: either 'none' or a list,
-	each item a pair of offsets (horizontal and vertical) from the element‘s box
-Animation type: by computed value
+Computed value: list, each item either 'none' or a pair of offsets
+  (horizontal and vertical) from the element‘s box
+Animation type: by computed value,
+  treating ''box-shadow-offset/none'' as ''0 0''
+  when interpolated with non-'none' values.
 </pre>
 
 <p>The 'box-shadow-offset' property defines one or more drop shadow offsets.
-The property accepts a comma-separated list of horizontal and vertical offset pairs,
+The property accepts a comma-separated list.
+Each item in that list can either be the ''box-shadow-offset/none'' value,
+which indicates no shadow, or a pair of horizontal and vertical offsets,
 where both values are described as <<length>> values.
 
 <dl>
+  <dt><dfn id="shadow-offset-none">none</dfn>
+  <dd>
+    No box shadow will be created.
+    Any other box shadow properties specified for that shadow in the list
+    of box shadows have no effect.
```

Comment:
Still seems confusing. Maybe something like

```suggestion
    The shadow will not be rendered.
    The values of other box shadow properties corresponding to this shadow have no effect.
```

Probably @fantasai can come up with something better.

Reply:
Note that I aligned that with the [`none` value for `animation-name`](https://drafts.csswg.org/css-animations-1/#valdef-animation-name-none).
And I believe it's clear enough that this means that other shadows in the list won't be affected by that.

Reply:
Ok, added your phrashing, though both seem equally good to me.